src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownEventListener.java

```diff
@@ -184,7 +182,7 @@ public void on(TownPreSetHomeBlockEvent event) {
 	@EventHandler
 	public void onDeleteTown(DeleteTownEvent event) {
 		if (SiegeController.hasSiege(event.getTownUUID()))
-			SiegeController.removeSiege(SiegeController.getSiegeByTownUUID(event.getTownUUID()), SiegeSide.ATTACKERS);
+			SiegeController.removeSiege(SiegeController.getSiegeByTownUUID(event.getTownUUID()));

```

Comment:
This looks like a Sieged town could delete itself to remove their town temporarily, deleting a nation's opportunity to get the warchest.

Reply:
Oh ye ok. Fixed.